src/main/java/com/jagrosh/jmusicbot/JMusicBot.java

```diff
@@ -111,6 +111,7 @@ public static void main(String[] args)
                         new SCSearchCmd(bot),
                         new ShuffleCmd(bot),
                         new SkipCmd(bot),
+                        new SeekCmd(bot),
```

Comment:
Commands should be listed in alphabetical order, so this should be above the shuffle command